compose/services.yml

```diff
@@ -76,6 +75,11 @@ services:
       BALENA_TOKEN_AUTH_ISSUER: api.${OPENBALENA_HOST_NAME}
       BALENA_TOKEN_AUTH_REALM: https://api.${OPENBALENA_HOST_NAME}/auth/v1/token
       COMMON_REGION:
+      REGISTRY2_CACHE_ENABLED: "false"
+      REGISTRY2_CACHE_ADDR: 127.0.0.1:6379
```

Comment:
why not `redis:6379` here?

Reply:
Registry has its own internal (ie in the same container) redis instance. When it was discussed, we determined we don't want to use the platform's redis instance for this sort of workload and reserve it for API-related functionality instead. Still, others might want otherwise and exposing a var for it is essentially free, so the env var was added that points by default to the inner redis instance.